core/typeguards.ts

```diff
@@ -0,0 +1,60 @@
+import is from '@sindresorhus/is';
+import { validate as uuidValidate } from 'uuid';
+import { version as uuidVersion } from 'uuid';
+
+export function assertNever(value: never, msg?: string): never {
+  throw new Error(isNonEmptyString(msg) ? msg : `Unexpected value: ${value}`);
+}
+
+export function isString(value: unknown): value is string {
+  return typeof value === 'string';
```

Comment:
Codacy has a fix for the issue: [Replace `'string'` with `"string"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7711626)
```suggestion
  return typeof value === "string";
```